bin/ota-declarations-validate.js

```diff
@@ -21,11 +18,10 @@ process.on('unhandledRejection', reason => {
 });
 
 program
-  .name('ota-validate-declarations')
+  .name('ota declarations validate')
   .description('Run a series of tests to check the validity of document declarations')
-  .version(version)
   .option('-s, --services [serviceId...]', 'service IDs of services to handle')
-  .option('-d, --documentTypes [documentType...]', 'document types to handle')
+  .option('-t, --termsTypes [termsType...]', 'terms types to handle')
   .option('-m, --modified', 'to only lint modified services already commited to git')
   .option('-so, --schema-only', 'only refilter exisiting snapshots with last declarations and engine\'s updates');
```

Comment:
The description of this command seems inaccurate 🤔 
```suggestion
  .option('-o, --schema-only', 'much faster check of declarations, but does not check that the documents are actually accessible.');
```